src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -347,31 +347,42 @@ export default class PhotoField extends React.Component {
               </button>}
             </div>
             <div className="cropper-control-container">
-              <div className="cropper-control-column">
+              <div className="cropper-control-row">
                 {smallScreen && <button className="cropper-control cropper-control-label-container va-button va-button-link" type="button" onClick={this.zoomOut}>
                   <span className="cropper-control-label">Make smaller</span>
                 </button>}
-                <button className={moveControlClass} type="button" onClick={this.moveUp}>
-                  <span className="cropper-control-label">Move up<i className="fa fa-arrow-up"></i></span>
-
-                </button>
-                <button className={moveControlClass} type="button" onClick={this.moveLeft}>
-                  <span className="cropper-control-label">Move left<i className="fa fa-arrow-left"></i></span>
-
-                </button>
-              </div>
-              <div className="cropper-control-column">
                 {smallScreen && <button className="cropper-control cropper-control-label-container va-button va-button-link" type="button" onClick={this.zoomIn}>
                   <span className="cropper-control-label">Make larger</span>
                 </button>}
-                <button className={moveControlClass} type="button" onClick={this.moveDown}>
-                  <span className="cropper-control-label">Move down<i className="fa fa-arrow-down"></i></span>
-
-                </button>
-                <button className={moveControlClass} type="button" onClick={this.moveRight}>
-                  <span className="cropper-control-label">Move right<i className="fa fa-arrow-right"></i></span>
-                </button>
               </div>
+              {[
```

Comment:
Codeclimate gave a repeating code error.

Reply:
@rianfowler You can feel free to ignore those in the future. We are planning to turn code climate off.

Reply:
Yeah @cehsu let me know. I was confused by the UI and thought I had to resolve all of these issues. 😂